src/plugin.py

```diff
@@ -30,6 +30,16 @@
 
 
 class BNetPlugin(Plugin):
+    async def get_game_time(self, game_id, context):
+        qpTimeMinutes = 0
+        log.info('game_id: ' + game_id)
+        if game_id == "5272175": # Overwatch
+            player_data = await self.backend_client.get_ow_player_data()
+            qpTime = re.search('0x0860000000000026.+?(?:(?P<h>\d+):)?(?P<m>\d+)', player_data.text)
```

Comment:
Private profiles will still render (my profile as example: https://playoverwatch.com/en-us/career/pc/elkangaroo-2885).
In that case `re.search` will return `None`, so should check `if qpTime is not None:` in this case